app/src/analytics/analyticsActions.js

```diff
@@ -77,6 +80,30 @@ export function trackSearchResultClicked(tilesetId, seriesgroup) {
   };
 }
 
+export function trackCreateFilterGroups(filterGroup) {
```

Comment:
do not hardcode filters. this should be fully dynamic, based on the header. If the layers change in a couple of weeks, we have to recode the GA tracking, which is a bad way to spend our time